modules/_labs/core/react/lib/CanvasProvider.tsx

```diff
@@ -7,6 +7,7 @@ import styled from '@emotion/styled';
 
 export interface CanvasProviderProps {
   theme: CanvasTheme;
+  includeInputProvider: boolean;
```

Comment:
Nit: I would maybe consider changing this to `disableInputProvider`, since that's generally how it's going to be used. For example, I wouldn't set the `disabled` attribute on an input component using an `enabled={false}` prop (that normally defaults to true)

Reply:
Can you add this prop to the readme?